org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/nls/NlsUtility.java

```diff
@@ -73,27 +75,27 @@ public static String bindText(String text, String... messageArguments) {
    * @param text
    *          The text to post-process. May be {@code null}.
    * @return The text with all post-processing applied.
-   * @see #postProcessText(Locale, String)
-   * @see #postProcessText(Locale, String, Collection)
+   * @see #postProcessText(Locale, String, String, Collection, String...)
    */
-  public static String postProcessText(String text) {
-    return postProcessText(NlsLocale.get(), text);
+  public static String postProcessText(@NlsKey String textKey, String text) {
+    return postProcessText(NlsLocale.get(), textKey, text);
```

Comment:
this overload is not necessary, is it? The one with varargs can also be used without arguments, right? Therefore this one could be deleted? And the varargs could be added to all other overloads?